agent/testflinger_agent/job.py

```diff
@@ -94,6 +95,16 @@ def run_test_phase(self, phase, rundir):
             )
             runner.register_stop_condition_checker(global_timeout_checker)
 
+        # We only need to check for output timeouts during the test phase
+        if phase == "test":
+            output_timeout_checker = OutputTimeoutChecker(
+                self.get_output_timeout()
+            )
+            runner.register_stop_condition_checker(output_timeout_checker)
+            runner.subscribe_event(
+                "output_received", output_timeout_checker.update
+            )
```

Comment:
Would be less error prone to define events somewhere (either as enum or just some global variables) then use them. Something like `RunnerEvents.output_received`

Reply:
Good idea, done.